src/dvbcsa_bs_avx2.h

```diff
@@ -166,5 +150,59 @@ static inline void block_sbox_avx3(dvbcsa_bs_word_t *src, dvbcsa_bs_word_t *dst)
 
 #define BLOCK_SBOX(in_buf, out_buf) block_sbox_avx2(in_buf, out_buf);
 
+/* block cipher 2-word load with byte-deinterleaving */
+// BS_LOAD_DEINTERLEAVE_8 replacement that only loads data that are already uninterleaved
```

Comment:
I would change this to
`/* BS_LOAD_DEINTERLEAVE_8 stub: deinterleaving is done by BLOCK_SBOX_PERMUTE */`